back/server/helpers/xmlHttpRequest.ts

```diff
@@ -0,0 +1,72 @@
+import http from 'http';
+import https from 'https';
+import env from './env';
+
+type RequestOptions = {
+  method: 'POST' | 'GET';
+  body?: string;
+  headers?: {
+    'Content-Type': 'application/json' | 'application/x-www-form-urlencoded';
+  };
+};
+type RequestResponse = {
+  ok: boolean;
+  status: number;
+  statusText: string;
+  body: string;
+};
+export const proxiedFetch = (url: string, options: RequestOptions): Promise<RequestResponse> => {
+  return new Promise((resolve, reject) => {
+    const uri = new URL(url);
+    const xmlHttpRequestOptions = {
+      hostname: uri.hostname,
+      port: uri.port || 443,
+      path: uri.pathname,
+      method: options.method,
+      headers: options.headers || {
+        'Content-Type': 'application/json',
+        'Cache-Control': 'no-store',
```

Comment:
The only difference with the UpSignOn-pro-server is this header, is this intended?

Reply:
I'm not sure we need this header actually. But I didn't want to challenge it here, so I kept it.